tests/test_errors.py

```diff
@@ -305,6 +305,18 @@ def finalize(self, cfg: Any) -> None:
         ),
         id="DictConfig[str,Color]:setitem_bad_value",
     ),
+    pytest.param(
+        Expected(
+            create=lambda: OmegaConf.structured(User),
+            op=lambda cfg: cfg.__setitem__("name", [1, 2]),
```

Comment:
Why not `cfg.name = [1,2]` ?

Reply:
Lambda doesn't allow that, and every other test uses setitem

Reply:
oh, right. forgot about that shit.